code/datums/mil_ranks.dm

```diff
@@ -28,15 +31,15 @@ var/global/datum/mil_branches/mil_branches = new()
 		var/datum/mil_branch/branch = branch_name
 		branch_name = initial(branch.name)
 	if(branch_name && branch_name != "None")
-		return branches[branch_name]
+		return LAZYACCESS(branches, branch_name)
```

Comment:
Why change these to LAZY? They're always set to list().

Reply:
Seems fixed by something, so removing LAZY.